pkgs/tools/misc/husky/default.nix

```
@@ -0,0 +1,22 @@
+{ lib, buildNpmPackage, fetchFromGitHub }:
+
+buildNpmPackage rec {
+  pname = "husky";
+  version = "8.0.3";
+
+  src = fetchFromGitHub {
+    owner = "typicode";
+    repo = "husky";
+    rev = "v${version}";
+    hash = "sha256-KoF2+vikgFyCGjfKeaqkC720UVMuvCIn9ApDPKbudsA=";
+  };
+
+  npmDepsHash = "sha256-u1dndTKvInobva+71yI2vPiwrW9vqzAJ2sDAqT9YJsg=";
+
+  meta = with lib; {
+    description = "Git hooks made easy woof";
```

```suggestion
    description = "Git hooks made easy 🐶 woof!";
```

Since I really missed that 🐶 emoji from the original description, I asked whether it's alright to use emojis in `meta.description`: https://matrix.to/#/!kjdutkOsheZdjqYmqp:nixos.org/$9Ppzf3264yieG3FCIsUgdoopnCppS-X1jVn_-VeHoRM 
There's not been a lot of response yet but at least some other packages already use emojis in their descriptions.

Also, another comment from @Tojnar regarding the specific husky description was:
> that is more like a tagline, the description should ideally describe what the package does

---

We don't add emojis to descriptions

---

Some have already found their way into nixpkgs:
https://github.com/NixOS/nixpkgs/blob/master/pkgs/tools/text/mdbook-emojicodes/default.nix#L26
https://search.nixos.org/packages?channel=unstable&show=mdbook-emojicodes&from=0&size=50&sort=relevance&type=packages

Is there actually a good reason not to have them, especially in cases where the description comes from upstream itself?

---

nix search for example displays those descriptions and not all terminals can render emojis which would lead to blank boxes or artefacts.